modules/apps/headless/headless-admin-user/headless-admin-user-impl/src/main/java/com/liferay/headless/admin/user/internal/resource/v1_0/UserGroupResourceImpl.java

```diff
@@ -83,6 +92,35 @@ public void deleteUserGroupUsers(Long userGroupId, Long[] userIds)
 			userGroupId, ArrayUtil.toArray(userIds));
 	}
 
+	@Override
+	public Page<UserGroup> getAssetLibrariesUserGroupsPage(
+			Long assetLibraryId, String search, Filter filter,
+			Pagination pagination, Sort[] sorts)
+		throws Exception {
+
+		DepotEntry groupDepotEntry = _depotEntryService.getGroupDepotEntry(
+			assetLibraryId);
+
+		return _getUserGroupsPage(
+			Collections.singletonMap(
```

Comment:
Maybe it would be great  to have here the delete action in order to take it into account when we will show the delete action for the user group list since we have to use it to filter the actions that user can do.